### ISSUE TITLE
[BUG] Inconsistency between license information

### ISSUE THREAD

**Issue by CarstenLeue** [docker/compose#12280]
### Description

The license information of the docker [compose source](https://github.com/docker/compose/blob/main/LICENSE) code is the Apache License. The information on the releases page for the [downloadable binaries](https://github.com/docker/compose/releases/download/v2.30.3/LICENSE) however is the docker license.

Is this on purpose? If so does this imply that one is allowed download the source code and build the binaries on their own using the Apache license? 

Purpose is to use the docker compose binaries outside of the scope of docker desktop.

### Steps To Reproduce

Consult

https://github.com/docker/compose/releases/download/v2.30.3/LICENSE
vs
https://github.com/docker/compose/blob/main/LICENSE

### Compose Version

```Text

```

### Docker Environment

```Text

```

**Comment by ndeloof** [comment]
> Is this on purpose? 

Yes. The source code is under Apache License - you can reuse it and build your own compose binaries - but the pre-built binaries as distributed under Docker End User License Agreement. While arguably confusing, this has no impact on using those regardless your usage of Docker Desktop.

**Comment by ndeloof** [comment]
https://github.com/docker/compose/blob/main/packaging/LICENSE should offer a link to "Docker End User License Agreement" for clarification

**Comment by ndeloof** [comment]
> are you suggesting that they are equivalent

nope, as Docker EULA only applies to binaries and is not an OSS license. I just mean using Compose binaries standalone doesn't count as a "seat" regarding Docker subscription licensing.
